pd-configuration-file.md

```diff
@@ -274,9 +274,18 @@ pd-server 相关配置项。
 
 ### `patrol-region-interval`
 
-+ 控制 replicaChecker 检查 Region 健康状态的运行频率，越短则运行越快，通常状况不需要调整
++ 控制 checker 检查 Region 健康状态的运行频率，越短则运行越快，通常状况不需要调整
 + 默认：10ms
 
+### `patrol-region-worker-count`
+
+> **警告：**
+>
+> 在当前版本中，将该配置项设置大于 1 时开启并发，为实验特性，不建议在生产环境中使用。
```

Comment:
```suggestion
> 将该配置项设置为大于 1 将启用并发检查。目前该功能为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
```

Reply:
Do we need to link it to pd?

Reply:
We can update the link to [issue](https://github.com/tikv/pd/issues) if we want users to report bugs in this repository